llm-tracing/main.py

```diff
@@ -0,0 +1,90 @@
+from fastapi import FastAPI
+from traceloop.sdk import Traceloop
+from dotenv import load_dotenv
+import os
+from pydantic import BaseModel
+from llama_index.core.agent.workflow import FunctionAgent
+from llama_index.llms.openai import OpenAI
+from opentelemetry.instrumentation.openai import OpenAIInstrumentor
+
+OpenAIInstrumentor().instrument()
+
+load_dotenv()
+
+# Ensure APP_NAME environment variable exists
+if not os.getenv("APP_NAME"):
+    raise ValueError("APP_NAME environment variable is not set")
+
+Traceloop.init(disable_batch=True, 
+               app_name=os.getenv("APP_NAME"))
```

Comment:
Is there any OTEL specific environment variable?

Reply:
`OTEL_SERVICE_NAME`

Reply:
https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/